x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/action_list_filters.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { memo, useCallback, useMemo } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiFilterGroup, EuiSuperUpdateButton } from '@elastic/eui';
+import type {
+  DurationRange,
+  OnRefreshChangeProps,
+} from '@elastic/eui/src/components/date_picker/types';
+import type { useGetEndpointActionList } from '../../../hooks';
+import type { DateRangePickerValues } from './action_list_date_range_picker';
+import { ActionListDateRangePicker } from './action_list_date_range_picker';
+import { ActionListFilter } from './action_list_filter';
+import type { FilterName } from './hooks';
+import { useTestIdGenerator } from '../../../hooks/use_test_id_generator';
+
+export const ActionListFilters = memo(
+  ({
+    dateRangePickerState,
+    isDataLoading,
+    onClick,
+    onChangeCommandsFilter,
+    onRefresh,
+    onRefreshChange,
+    onTimeChange,
+  }: {
+    dateRangePickerState: DateRangePickerValues;
+    isDataLoading: boolean;
+    onChangeCommandsFilter: (selectedCommands: string[]) => void;
+    onRefresh: () => void;
+    onRefreshChange: (evt: OnRefreshChangeProps) => void;
+    onTimeChange: ({ start, end }: DurationRange) => void;
+    onClick: ReturnType<typeof useGetEndpointActionList>['refetch'];
+  }) => {
+    const getTestId = useTestIdGenerator('response-actions-list');
+    const filters = useMemo(() => {
+      // TODO: add more filter names here (Users, Hosts, Statuses)
+      const filterNames: FilterName[] = ['Actions'];
```

Comment:
Are these value in the array visible on the UI?
If so, they need to be i18n'd

Reply:
done 22e0c598c6bdfbb92de18cd6cc7d427c8dae32f9